configs/vm_resource.py

```diff
@@ -29,6 +29,14 @@ class Project(enum.Enum):
   TPU_PROD_ENV_AUTOMATED = "tpu-prod-env-automated"
 
 
+class ImageProject(enum.Enum):
+  DEEP_LEARNING_PLATFORM_RELEASE = ("deeplearning-platform-release",)
```

Comment:
Is there special requirement for Image project so that we need tuple of strings?

Reply:
Oh, thanks for catching the typo. That's why all the GPU tests failed.